apps/dashboard/app/models/project.rb

```diff
@@ -53,7 +64,50 @@ def save!
     true
   end
 
+  def update(attributes)
+    @title          = attributes[:title] if attributes[:title]
+    @description    = attributes[:description] if attributes[:description]
+    @icon           = attributes[:icon] if attributes[:icon]
+
+    if self.valid?
+      write_manifeset
+      true
+    else
+      false
+    end
+  end
```

Comment:
I think see now we shouldn't be following how products do it. We should `delegate` to manifest directly for stuff, or at least do it outright like 
```ruby
def description
  manifest.description
end
```

We have a lot of member variables that are actually parts of the manifest which isn't good encapsulation - and I see that's a pattern in `app/models/products.rb`.  I think a better example may be `app/apps/ood_app.rb`.

In either case if the manifest is the _source of truth_ for all this information - Isn't it as simple as this? 
```suggestion
  def update(attributes)
    m = Manifest.load(manifest_path).merge(attributes)
    m.valid? ? false : m.save(manifest_path)
  end
```